README.md

```diff
@@ -112,7 +112,8 @@ Signed,
 - Dorian Peregrim ([@DorianPeregrim](https://github.com/DorianPeregrim)), PHP Developer, Russia 🇷🇺
 - Dzmitry Bekish ([@SofWar](https://github.com/sofwar)), Full Stack Developer, Belarus
 - Eder Soares ([@edersoares](https://github.com/edersoares)), Software Engineer, Brazil 🇧🇷
-- Egor Rudkov ([@gren236](https://github.com/gren236)), Backend Developer, Russia
+- Eduardo Dias ([@edurodriguesdias](https://github.com/edurodriguesdias)), Software Engineer, Brazil 🇧🇷
+- Egorza Rudkov ([@gren236](https://github.com/gren236)), Backend Developer, Russia
```

Comment:
Hello, there seems to be another name change here. Can you please the Egorza typo back to "Egor"?

Reply:
@petk done ✅